### ISSUE TITLE
Framework not compiling for x86_64/i386 by default

### ISSUE THREAD

**Issue by jamescs** [smartdevicelink/sdl_ios#257]
SDL doesn't include all architectures in the compiled framework (x86_64 and i386). Can we update the Build Settings to include all architectures by default?

**Comment by jamescs** [comment]
(when building from Xcode, that is.

**Comment by joeljfischer** [comment]
Was this resolved for you?

**Comment by joeljfischer** [comment]
So that you know, the reason that I chose not to compile for i386 is that it would have required a separate build, with a build script that lipo the two together. Unfortunately, an app store submission bug prevents apps using fat binaries like this from being properly submitted. [See this radar](http://www.openradar.me/radar?id=6409498411401216) for more detail.

There are workarounds, but they require the developer to follow additional specific steps. I decided it wasn't worth the headache at this point.

I'm hopeful that the app thinning tech debuting with iOS 9 will fix the situation, but it may introduce it's own problems, we don't know yet.
